packages/design-system/src/connect/components/search-bar/search-bar.test.tsx

```diff
@@ -25,15 +30,19 @@ const filterItems: ConnectSearchBarProps["filterItems"] = [
 ];
 
 describe("ConnectSearchBar Component", () => {
-  it("should match snapshot", () => {
+  it("should match snapshot", async () => {
     const { asFragment } = render(
       <ConnectSearchBar
         filterItems={filterItems}
         filterLabel="File type"
         onChange={() => {}}
         placeholder="Search Files"
         value="test"
-      />,
+      />
+    );
+
+    await waitForElementToBeRemoved(() =>
```

Comment:
Snapshots that have icons need to way for them be loaded. Maybe there is a more general way to do this in storybook?